tensorflow/python/estimator/keras_test.py

```diff
@@ -477,14 +477,8 @@ def test_custom_objects(self):
     keras_mobile = mobilenet.MobileNet(weights=None)
     keras_mobile.compile(loss='categorical_crossentropy', optimizer='adam')
     custom_objects = {
-        'relu6': mobilenet.relu6,
         'DepthwiseConv2D': mobilenet.DepthwiseConv2D
     }
-    with self.assertRaisesRegexp(ValueError, 'relu6'):
```

Comment:
Instead of removing this, should it just expect "DepthwiseConv2D" in the exception?

Reply:
Thanks! I fixed it.

Reply:
`DepthwiseConv2D` is also no longer custom objects. So mobilenet is not suitable for `test_custom_objects`. I'll fix it to define simple model in `keras_test.py`.

Reply:
Ack. Please fix the test case.